frontend/src/app/app.module.ts

```diff
@@ -20,6 +20,12 @@ import { ArtshowComponent } from './artshow/artshow.component';
 import { ArtistComponent } from './artshow/artist/artist.component';
 import { DealersComponent } from './dealers/dealers.component';
 import { DealerComponent } from './dealers/dealer/dealer.component';
+import { TabletopComponent } from './gaming/tabletop/tabletop.component';
+import { LarpComponent } from './gaming/larp/larp.component';
+import { VideoComponent } from './gaming/video/video.component';
```

Comment:
It's weird that this is something VideoComponent and sometimes VideoGameComponnent. Might it make more sense to have it just be ./gaming/videogames even though it's redundant? Just so that it's consistent?

Reply:
i only changed it in the main routing so that it wasn't horribly confusing there - and i went back and forth about changing it after i first instantiated things, but ended up not doing that. I can get rid of the clarification in the routing :)